src/Storage.java

```diff
@@ -0,0 +1,96 @@
+import java.io.*;
+
+/**
+ * Storage class that handles the reading and writing process from a designated text file,
+ * referred to as a save file.
+ */
+public class Storage {
+    public final String file;
+
+    /**
+     * Constructor that creates a Storage object.
+     * @param file the file task sessions will be saved in.
+     */
+    public Storage(String file) {
+        this.file = file;
+    }
+
+    /**
+     * Writes the current TaskList to a save file.
+     * @param taskList TaskList containing Tasks to write.
+     * @param ui the Ui associated to the current Duke object.
+     */
+    public void saveTasks(TaskList taskList, Ui ui) {
+        File savedFile = new File(".//SAVED-TASKS.txt");
+        boolean exists = savedFile.exists();
```

Comment:
Boolean variable can be renamed to something like "isFound" or "doesExist".

Reply:
Yes, I agree with zhuoyao, it sounds more like booblean variable